awx/api/generics.py

```diff
@@ -88,15 +88,30 @@ def get(self, request, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         ret = super(LoggedLoginView, self).post(request, *args, **kwargs)
+
+        client_addr = request.META.get('REMOTE_ADDR', None)
+
+        # If an authorized header contains an IP, override the client_addr
+        for custom_header in settings.REMOTE_HOST_HEADERS:
+            for value in request.META.get(custom_header, '').split(','):
+                value = value.strip()
```

Comment:
I don't like modifying the loop iterator, `value`, directly, especially if you're not going to use it. Unless `request` object has a special get that makes copies, you're not getting a value, you're getting a reference and if you change it it will change in the `request` object, too. And that might be kind of unexpected, no? It can certainly cause gotchas. EDIT: OK, I now notice that you're using `split()` and I guess that makes new objects. However, beyond that I don't understand the use of `split(',')` here or how we know that the first value will be the right `client_addr`.